index.js

```diff
@@ -21,6 +21,10 @@ if (!heimdall.hasMonitor('async-disk-cache')) {
   heimdall.registerMonitor('async-disk-cache', function AsyncDiskCacheSchema() {});
 }
 
+var homedir = require('os').homedir().split(path.sep);
+var username = homedir[homedir.length - 1];
```

Comment:
Is `process.env.USER` guaranteed to be set? If so, seems good. Otherwise, maybe use it and fallback to the path splitting?